applications/StructuralMechanicsApplication/python_scripts/structural_response.py

```diff
@@ -7,31 +7,45 @@
 
 import time as timer
 
+# ==============================================================================
 class ResponseFunctionBase(object):
     """The base class for structural response functions. Each response function
     is able to calculate its response value and gradient.
     All the necessary steps have to be implemented, like e.g. initializing,
     solving of primal (and adjoint) analysis ...
     """
 
+    def RunCalculationProcess(self, calculate_gradient):
```

Comment:
I would go for ```Run``` because CalculationProcess does not add any information.
Also that is more in line with the analysis/stage